test/e2e/network/service.go

```diff
@@ -147,6 +192,7 @@ func checkAffinity(cs clientset.Interface, execPod *v1.Pod, serviceIP string, se
 			return false
 		}
 		if !trackerFulfilled {
+			serviceIPPort := net.JoinHostPort(serviceIP, strconv.Itoa(servicePort))
 			checkAffinityFailed(tracker, fmt.Sprintf("Connection to %s timed out or not enough responses.", serviceIPPort))
```

Comment:
(you could just remove `serviceIPPort` from the error message... it's obvious from context)

Reply:
done